console: Improve console quickstart docs (#916)
redpanda-data/documentation: docs/platform/quickstart/console-installation.mdx

</head>

## Quickstart

### Redpanda platform with Docker Compose

> The easiest way to get started with Redpanda Console on a local machine is to start Redpanda platform
> with Docker compose. Refer to the [Docker Compose page](../console/reference/docker-compose.mdx) to
> launch Redpanda platform locally.

Reviewer: ```suggestion
The easiest way to get started with Redpanda Console on a local machine is to start Redpanda platform
with Docker compose. Refer to [Docker Compose](../console/reference/docker-compose.mdx) to
launch Redpanda platform locally.
```

### Connect to a remote cluster

> To connect Redpanda Console to a Kafka cluster that does not run on the same machine where you want
> to run Console, use this command:

Reviewer: ```suggestion
To connect Redpanda Console to a Kafka cluster that is not running on the same machine where you want
to run Console:
```

  docker.redpanda.com/vectorized/console:latest
```

:::note
Refer to the [Console reference config](../console/reference/config.mdx) for a full list of configurations.
:::

### Connect to a local cluster

To connect Redpanda Console to a Kafka cluster that is running on your local machine
or in another Docker container:

```shell
docker run --network=host \
  -e KAFKA_BROKERS=localhost:9092 \
  docker.redpanda.com/vectorized/console:latest
```

> Note that only the initial connection is made through the `KAFKA_RBOKERS` address you specified.

Reviewer: ```suggestion
Note that only the initial connection is made through the `KAFKA_BROKERS` address you specified.
```

> After the initial connection is established, your client application will talk to the cluster to
> get all broker' addresses so that it can communicate to all brokers directly. Each broker returns
> the configured `advertised_listener` address that matches the port the initial connection has been

Reviewer: ```suggestion
After the initial connection is established, your client application communicate with the cluster to
get all broker addresses so that it can communicate with all brokers directly. Each broker returns
the configured `advertised_listener` address that matches the port to which the initial connection has been made.
```

> made to. Make sure that this address is reachable by your client application.

Reviewer: ```suggestion
made. Ensure that this address is reachable by your client application.
```

:::note
The host networking driver only works on Linux hosts, and is not supported on Docker Desktop for Mac, 

> Docker Desktop for Windows, or Docker EE for Windows Server. An alternative way is to use the right
> binary for your OS and run this binary instead. We provide [binaries](#binaries-and-linux-packages) 
> for all major operating systems and architectures.

Reviewer: ```suggestion
Docker Desktop for Windows, or Docker EE for Windows Server. An alternative way is to use the correct
binary for your OS and run it instead. Redpanda provides [binaries](#binaries-and-linux-packages) 
for all major operating systems and architectures.
```

  docker.redpanda.com/vectorized/console:latest
```

:::note

> The full list of configurations can be found in the [Console reference config](../console/reference/config.mdx).

Reviewer: ```suggestion
Refer to the [Console reference config](../console/reference/config.mdx) for a full list of configurations.
```

:::

### Connect to a local cluster

To connect Redpanda Console to a Kafka cluster that is running on your local machine

> or in another Docker container run:

Reviewer: ```suggestion
or in another Docker container:
```

```shell
docker run --network=host \
  -e KAFKA_BROKERS=localhost:9092 \
  docker.redpanda.com/vectorized/console:latest
```

> Take note that only the initial connection is made through your provided `KAFKA_RBOKERS` address.

Reviewer: ```suggestion
Note that only the initial connection is made through the `KAFKA_RBOKERS` address you specified.
```
